ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PerformanceEnvelope.scala

```diff
@@ -37,7 +37,7 @@ import scala.util.{Failure, Success, Try}
 import com.daml.ledger.test.performance.{PingPong => PingPongModule}
 import org.slf4j.Logger
 
-case class Envelope(name: String, transactionSizeKb: Int, throughput: Int, latencyMs: Int)
+sealed case class Envelope(name: String, transactionSizeKb: Int, throughput: Int, latencyMs: Int)
```

Comment:
`case class`es should never be inherited. If anything, I'd recommend making this `final` rather. `sealed` makes little sense.

Reply:
Right, the enum-avoiding (sigh) "recommended path" is now `sealed trait` I believe.

Reply:
Either that, or simply make `Envelope` a `sealed abstract class`. Do you really need it to be a `case class`? The reason why `case class`es should be `final` in vast majority of cases (no pun intended) is to [make sure equality makes sense](https://stackoverflow.com/questions/11158929/what-is-so-wrong-with-case-class-inheritance/11159036#11159036). With [the notable exception of `sealed abstract case class`es](https://gist.github.com/tpolecat/a5cb0dc9adeacc93f846835ed21c92d2).

Reply:
It was a leftover when the code was ported. Aligned with the recommended pattern.